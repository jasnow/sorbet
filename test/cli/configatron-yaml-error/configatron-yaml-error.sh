#!/bin/bash

main/sorbet --silence-dev-message --configatron-file=test/cli/configatron-yaml-error/configatron-yaml-error.yaml test/cli/configatron-yaml-error/configatron-yaml-error.rb 2>&1
