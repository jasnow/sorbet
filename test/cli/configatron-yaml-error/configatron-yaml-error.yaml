default: &default {}
  test_bool: true
