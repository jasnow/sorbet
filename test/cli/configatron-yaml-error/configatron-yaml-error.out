???: YAML syntax error parsing file test/cli/configatron-yaml-error/configatron-yaml-error.yaml: end of map not found https://srb.help/4020
Errors: 1
